src/examples/05-custom-dialog.html

```diff
@@ -1,89 +0,0 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
I think maybe you underestimate the value of examples in learning. I think it would be well-worth timeboxing 15 minutes to update this example to use the new API and to then link to the example from the documentation. Examples are basically the only way some people learn :)

Thoughts?